hipe4ml/tree_handler.py

```diff
@@ -0,0 +1,269 @@
+"""
+Simple module with a class to manage the data used in the analysis
+"""
+
+import uproot
+import numpy as np
+
+
+class TreeHandler:
+    """
+    Class for storing and managing the data of a ROOT tree
+    """
+
+    def __init__(self, file_name, tree_name, columns_names=None):
+        """
+        Open the file in which the selected tree leaves are converted
+        into pandas dataframe columns
+
+        Parameters
+        ------------------------------------------------
+        file_name: str
+            Name of the input file where the data sit
+        tree_name: str
+            Name of the tree within the input file
+        columns_name: list
+            List of the names of the branches that one wants to analyse
+        """
+        self._file = uproot.open(file_name)
```

Comment:
It would be nice to have the possibility to load parquet files besides root files (for example @fcatalan92 and I typically work with parquet files in input). You could for example check the extension of the input file and use `uproot` in case of `ROOT` files and `pandas.read_parquet` in case of `parquet` files